docs/topics/using-openrewrite-recipes.adoc

```diff
@@ -22,7 +22,7 @@ For example, the OpenRewrite recipe `org.jboss.windup.JavaxToJakarta` renames im
 +
 [source,terminal,subs="attributes+"]
 ----
-$ ./{mta-cli} transform openrewrite --input </path/to/source/project> \
```

Comment:
This is the correct command

Reply:
Fixed in PR - https://github.com/windup/windup-documentation/pull/828